### ISSUE TITLE
tutorial and help (Dialog?) window improvements

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/Panoptes-Front-End#2961]
## Expected behavior
- [ ] Escape closes window
- ~~Window draggable~~ EDIT: pass, per Sarah's comment, not typical behaviour
- [ ] Window dynamically sized based on media content

## Other Solutions to this Issue
Consider using tools differently, i.e. Tutorial not big enough? Probably should be using Help instead. Or consider Field Guide.

**Comment by srallen** [comment]
What is meant by 'window draggable'? If it is what I think you mean, then that isn't typical behavior of dialogs. Perhaps a dialog is being somewhere that should be something else?
